modules/loader.js

```diff
@@ -1054,13 +1041,13 @@ Loader.prototype.findGoodPeers = function(peers) {
 
 // Public methods
 /**
- * Gets a list of "good" peers from network.
```

Comment:
Please extend the comment and explain what does it mean to be a good peer.

Reply:
A good peer is what `findGoodPeers()` decides. I haven't modified that function. It performs a complex calculation that cannot be explained easily:
- It filters unreachable peers and those ones below the node's height itself.
- It orders peers by descending height
- It creates a histogram of peer's heights
- It removes outliers

I'd open another PR if we need to review that process.